### PR TITLE
DEVDOCS-5896: [Update] add cart redirect URL

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5896]

## What changed?
Add cart redirect URL

## Release notes draft

* 

## Anything else?

ping {names}

[DEVDOCS-5896]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx [hunk 1]
```diff
@@ -530,6 +531,48 @@ If using the getCart query with customer impersonation tokens, you must include
   </Tab>
 </Tabs>
 
+### Create a cart redirect URL
+
+<Callout type="info">
+You can use a cart redirect URL for embedded checkouts or to redirect your checkout. A cartEntity ID is required. To create a cartEntity ID, use the [create a cart](#Create-a-cart-using-a-simple-product) mutation.
```
Comment (bc-andreadao) [inline docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx:537]:
```suggestion
You can use a cart redirect URL for embedded checkouts or to redirect your checkout. You need a `cartEntity` ID. To create a `cartEntity` ID, use the [create a cart](#Create-a-cart-using-a-simple-product) mutation. 
```
active voice

**Review by bc-andreadao — APPROVED** [review]

File: docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx [hunk 2]
```diff
@@ -530,6 +531,48 @@ If using the getCart query with customer impersonation tokens, you must include
   </Tab>
 </Tabs>
 
+### Create a cart redirect URL
+
+<Callout type="info">
+You can use a cart redirect URL for embedded checkouts or to redirect your checkout. You need a `cartEntity` ID. To create a `cartEntity` ID, use the [create a cart](#Create-a-cart-using-a-simple-product) mutation.
```
Comment (bc-tgomez) [inline docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx:537]: Content under the callout tag needs to be moved one (to the right).

File: docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx [hunk 3]
```diff
@@ -530,6 +531,48 @@ If using the getCart query with customer impersonation tokens, you must include
...
+           }
+       }
+  
+   }
+}
+```
+  </Tab>
+  <Tab>
+    
+```json filename="Example mutation: Create cart redirect URL" showLineNumbers copy
```
Comment (bc-tgomez) [inline docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx:559]: This code block needs to be moved one (to the right). You had it right on [#L542](https://github.com/bigcommerce/docs/pull/289/files#diff-6ff391d937e95e1d2821796b1c639c1ca0fb9f698e63a570ff895188a2c6d7b5R542)

**Review by bc-tgomez — COMMENTED** [review]
Small format changes but looks great!

**Review by bc-tgomez — APPROVED** [review]
Awesome! Approved!

**Pull request merged**